#DoCoMo Dialogue API Response message
Header header
string code 
string textForDisplay 
string textForSpeech 
DoCoMoQaAnswer[] answer
